### ISSUE TITLE
Consider maybe Obsidian

### ISSUE THREAD

**Issue by userrand** [sindresorhus/awesome-electron#732]
This is a great list. I believe Obsidian is also electron based (closed source) and it is quite popular (I could be wrong but it seems to be the second most favorited note app after Notion). For example, the obsidian subreddit has 107 000 members https://www.reddit.com/r/ObsidianMD/

**Comment by ivanmarribas** [comment]
Yes, it uses Electron, as listed in the [credits](https://help.obsidian.md/Obsidian/Credits#Electron).

**Comment by sindresorhus** [comment]
PR welcome
